tests/ludwig/features/test_binary_feature.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict
+
+import pytest
+import torch
+
+from ludwig.features.binary_feature import BinaryInputFeature
+
+SEQ_SIZE = 2
+BINARY_W_SIZE = 1
+
+
+@pytest.fixture(scope="module")
+def binary_config():
+    return {
+        "name": "binary_feature",
+        "type": "binary",
+    }
+
+
+@pytest.mark.parametrize("encoder", ["passthrough"])
+def test_binary_input_feature(binary_config: Dict, encoder: str) -> None:
+    binary_config.update({"encoder": encoder})
+    binary_input_feature = BinaryInputFeature(binary_config)
+    binary_tensor = torch.randn([SEQ_SIZE, BINARY_W_SIZE], dtype=torch.float32)
+    encoder_output = binary_input_feature(binary_tensor)
+    assert encoder_output["encoder_output"].shape[1:] == binary_input_feature.encoder_obj.output_shape
```

Comment:
Why not just `binary_input_feature.output_shape`? Perhaps we need to change `binary_input_feature.output_shape` to return the encoder output shape?